easybuild/easyconfigs/r/REMORA/REMORA-1.8.2-goolf-1.7.20.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'PackedBinary'
+
+name = "REMORA"
+version = "1.8.2"
+
+homepage = 'https://github.com/TACC/remora'
+description = """REsource MOnitoring for Remote Applications"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/TACC/remora/archive/']
+sources = ['v%(version)s.tar.gz']
+
+unpack_options = '--strip-components=1'
```

Comment:
the intentions are good here, but this has a side-effect that the `install.sh` script is being run three times, once for each subdirectory in the unpacked tarball (see `install_step` in `PackedBinary`...)

better approach: drop `unpack_options` and use `cd remora-%(version)s && ...` in `install_cmd`